auth-service/src/main/java/com/artsem/api/authservice/broker/EmailVerificationMessage.java

```diff
@@ -0,0 +1,7 @@
+package com.artsem.api.authservice.broker;
+
+import lombok.Builder;
+
+@Builder
+public record EmailVerificationMessage(String email, String token) {
+}
```

Comment:
Для лучшей читаемости следует размещать параметры на отдельных строках.
```suggestion
@Builder
public record EmailVerificationMessage(
        String email,
        String token) {
}
```